docs/source/daml/daml-studio.rst

```diff
@@ -223,6 +223,10 @@ Missing authorization on create
 If a contract is being created without approval from all authorizing
 parties the commit will fail. For example:
 
+.. literalinclude:: daml-studio/daml/CreateAuthFailure.daml
+  :language: daml
+  :lines: 7-13,15-18
+
```

Comment:
This seems like duplicating the same lump twice?

Reply:
testing something and mistakenly left this in - gone now